packages/amplify_flutter/lib/amplify.dart

```diff
@@ -60,14 +63,14 @@ class AmplifyClass extends PlatformInterface {
     if (!isConfigured) {
       try {
         if (plugin is AuthPluginInterface) {
-          Auth.addPlugin(plugin as AuthPluginInterface);
+          Auth.addPlugin(plugin);
```

Comment:
this shouldn't be needed if the library is properly bootstrapped, I struggled with this as well, but eventually realized it's not necessary.

Reply:
fixed.